### ISSUE TITLE
Scroll page on mobile

### ISSUE THREAD

**Issue by lol7344** [spacedeck/spacedeck-open#43]
When a "space" is opened on mobile, the user is unable to scroll through the page by using two fingers. Instead, the two fingers are registered as the same one and the cursor keeps bouncing from finger to finger, messing up everything.

**Comment by mntmn** [comment]
This is true, it behaves kinda correctly in web inspector in chrome when simulating iPad/iPhone, but it doesn't work on the actual device (iPhone 7 Mobile Safari in my case).

**Comment by mntmn** [comment]
This is fixed now in 8e11b6c4faffd2b95447d67c3f8736e34b748126. You can now scroll with one finger when using touch. Pinch-to-zoom is not implemented at the moment, and deselection on touching empty space doesn't work yet.
